html/resources.html

```diff
@@ -0,0 +1,60 @@
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="description" content="This is a web site for newbie amateur astronomers." />
+    <title>What Can I See Tonight?</title>
+
+    <link rel="stylesheet" href="../css/reset.css" type="text/css" />
+    <link rel="stylesheet" href="../css/styles.css" type="text/css" />
+    <link href='http://fonts.googleapis.com/css?family=Roboto:400,400italic,700' rel='stylesheet' type='text/css'>
+  </head>
+  <body>
+      <div class="container clearfix">
+    <header>
+      <div class="container clearfix">
+            <h1>What Can I See Tonight?</h1>
+      </div>
+      <div class="container-menu clearfix">
+        <ul>
+          <li><a href="index.html">Main</a></li>
+          <li><a href="blog.html">Blog</a></li>
+          <li><a href="resources.html">Resources</a></li>
+          <li><a href="about.html">About</a></li>
+        </ul>
+      </div>
+
+    </header>
+
+    <main>
+      <h2>Resources</h2>
+
+      <p>There are many helpful web sites and apps for astronomers. Below is a partial list:</p>
+
+      <ul>
+        <li><a href="http://www.skyandtelescope.com/">Sky and Telescope</a></li>
+        <li><a href="http://www.astronomy.com/">Astronomy Magazine</a></li>
+        <li><a href="http://nightsky.jpl.nasa.gov/"</a>NASA Night Sky Network</a></li>
+        <li><a href="http://www.fifthstarlabs.com/#sky-guide"</a>Sky Guide (iOS)</a></li>
+        <li><a href="http://google-sky-map.en.softonic.com/android/download"</a>Google Sky Map (Android)</a></li>
+      </ul>
+
+    </main>
+
+  <aside>
+          <h2>What Can Hubble See?</h2>
+          <img src="../images/hubble_friday_08072015.jpg" alt="NGC 6818">
+          <p>The planetary nebula NGC 6818, also known as the Little Gem Nebula, which is just over half a light-year across. <a href="http://www.nasa.gov/mission_pages/hubble/multimedia/index.html?id=368246">(image courtesy of NASA)</a></p>
+          <!--Some kind of RSS feed/link to cool NASA images would be ideal. Could also just use a rotating selection of 10-15 saved images/captions that are randomly chosen via js.-->
+  </aside>
+
+      <footer>
+        <div class = "container clearfix">
+            <p>Questions? Contact <a href="#">Jen Stevens.</a></p>
```

Comment:
make sure you style all the `<a>` tags on your page. It's super important that links have a visual cue to a user that they are, indeed, links.